template/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We can modify /template/.template.config/template.json to exclude this file if logging is disabled - something like:
```
...
    "sources": [
        {
            "modifiers": [
                {
                    "condition": "(disable-health-check)",
                    "exclude": [
                        "HealthChecks/**"
                    ]
                },
                {
                    "condition": "(disable-logging)",
                    "exclude": [
                        "appsettings.json"
                    ]
                }
            ]
        }
    ]
```

Reply:
done and tested